src/jit/lsraarmarch.cpp

```diff
@@ -403,7 +403,17 @@ void Lowering::TreeNodeInfoInitPutArgReg(
     info.srcCount++;
 
     // Set the register requirements for the node.
-    const regMaskTP argMask = genRegMask(argReg);
+    regMaskTP argMask = genRegMask(argReg);
```

Comment:
I see. We may need to modify that in future, because once we are doing the containment analysis in `Lowering`, and then doing `TreeNodeInfoInit` on the fly, we need to be able to set all the counts and candidates in a forward walk (without modifying a node's info from its parent). But this works for now.